cdap-ui/app/features/hydrator/controllers/create/toppanel-ctrl.js

```diff
@@ -44,6 +44,14 @@ angular.module(PKG.name + '.feature.hydrator')
       this.metadataExpanded = false;
     };
 
+    this.onEnterOnMetadata = function(event) {
+      if (event.keyCode === 13) {
```

Comment:
nit - have a comment that describes which keys '13' and '27' correspond to.
otherwise LGTM